src/app/messages/en/messages.json

```diff
@@ -116,5 +116,7 @@
   "i18n:cta:text": "Read, search, highlight, add notes, build study guides -- all for <strong>free</strong>.",
   "i18n:cta:button:signup": "Sign up",
   "i18n:cta:button:separator": "or",
-  "i18n:cta:button:login": "Log in"
+  "i18n:cta:button:login": "Log in",
+  "i18n:toolbar:studyguides:popup:close-button:aria-label": "Close Study Guides",
+  "i18n:toolbar:studyguides:popup:heading": "Study Guide"
```

Comment:
There is already one text related to study guides: `i18n:toolbar:study-guides:text`
I've added it after the `i18n:toolbar:search:toggle` to keep it in alphabetic order but these keys are actually not in alphabetic order anyway so you can just decide where to put these keys but group them together with consistent names